aws-iam-role-ci-user-manager/main.tf

```diff
@@ -0,0 +1,66 @@
+data "aws_caller_identity" "current" {}
+
+module assume_role_policy {
+  source             = "../aws-assume-role-policy"
+  source_account_ids = toset([data.aws_caller_identity.current.account_id])
+}
+
+resource "aws_iam_role" "ci-manager" {
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing version constraint for provider "aws" in "required_providers"